src/Controls/src/Core/PublicAPI/net-android/PublicAPI.Unshipped.txt

```diff
@@ -68,6 +68,7 @@ override Microsoft.Maui.Controls.Handlers.Compatibility.TableViewModelRenderer.G
 override Microsoft.Maui.Controls.Handlers.Compatibility.TableViewRenderer.OnMeasure(int widthMeasureSpec, int heightMeasureSpec) -> void
 override Microsoft.Maui.Controls.Handlers.Items.MauiCarouselRecyclerView.OnAttachedToWindow() -> void
 override Microsoft.Maui.Controls.Handlers.Items.MauiCarouselRecyclerView.OnDetachedFromWindow() -> void
+static Microsoft.Maui.Controls.TabbedPage.MapUpdateOffscreenPageLimit(Microsoft.Maui.Handlers.ITabbedViewHandler! handler, Microsoft.Maui.Controls.TabbedPage! view) -> void
```

Comment:
You can delete this one now, which should fix the compile errors

Reply:
@PureWeen I had deleted this code. thanks